source/SpinalHDL/Data types/bool.rst

```diff
@@ -115,13 +124,21 @@ Logic
     * f := req || (f && !ack)
     */
 
-  // mind the order of assignments!
+  // mind the order of assignments!  last one wins
   val g = RegInit(False) setWhen(req) fallWhen(ack)
   // equivalent to g := ((!g) && req) || (g && !ack)
 
 Edge detection
 ~~~~~~~~~~~~~~
 
+This feature will always imply a :ref:`RegNext <regnext>` pattern to track the state history
```

Comment:
```suggestion
All edge detection functions will instantiate a register via :ref:`RegNext <regnext>` to get a delayed value of the ``Bool`` in question.
```
?

Reply:
`...instantiate an additional register via ...` ?  to clarify the register being talked about ?   It may sound verbose to us to restate that but I ask myself how the documentation reads after it has been put through an automated translator into another natural language.

Reply:
Not sure google translate is a good test for this... but yes, let's go for "an additional register via"